perllib/FixMyStreet/App/Controller/My.pm

```diff
@@ -57,6 +57,7 @@ sub planned : Local : Args(0) {
     $c->forward('/reports/stash_report_sort', [ 'shortlist' ]);
     $c->forward('get_problems');
     if ($c->get_param('ajax')) {
+        $c->stash->{shortlist} = (!defined $c->get_param('sort') || $c->get_param('sort') eq 'shortlist');
```

Comment:
You can just use the stash->{sort_key} here (which has already done param checking for you).